install/jsx.py

```diff
@@ -22,8 +22,8 @@ def load_install_tools():
     if not os.path.exists(path):
         rootdir = os.path.dirname(os.path.abspath(__file__))
         path = os.path.join(rootdir, "InstallTools.py")
-
-        if not os.path.exists(path) or path.find("/code/") == -1:
+        print
+        if not os.path.exists(path) and path.find("/code/") == -1:
```

Comment:
This has a side effect that you don't always get latest development/branch code. You might be running stale code from an earlier installation

Reply:
is correct, the thing is that the doc for running it from code is not in line with the code but you are right, there must be a better way

Reply:
I also wonder what the original intention of that code was then

Reply:
needs to be or, because idea is if you have /code/ then you run it from a repo which means you have updated yourself, when it doesn't exist ofcourse download needs to happen

Reply:
if /code/ is there it means all is run from same repo so all in sync